src/service/platform-impl.js

```diff
@@ -98,6 +98,15 @@ export class Platform {
   }
 
   /**
+   * Whether the current browser is isStandAlone.
+   * @return {boolean}
+   */
+  isStandAlone() {
```

Comment:
Since the `navigator.standalone` property is all lowercase, we should only capitalize the first letter of `Standalone` where it's used in camel case e.g. `isStandalone`